libs/profiles/src/lib.rs

```diff
@@ -9,6 +9,7 @@ enum CpuOptLevel {
     none,
     native,
     neon_v8,
+    x86_64_v1,
```

Comment:
To match the rust target CPU patterns this should just be `x86_64`

Reply:
I'd rather name it something like `x86_64_obsolete` or `x86_64_legacy` to indicate to users or deployers that this is probably not the right choice for them.